examples/cross-build/Hello.js.scala

```diff
@@ -1,5 +1,5 @@
-// using scala 2.13
-// using target scala-js
+//> using scala 2.13
+//> using target scala-js
```

Comment:
Did I omit a moment when we allowed strings without quotes?

Reply:
I think we do not check examples if compile file, a good catch, I've open a new issue for that: https://github.com/VirtusLab/scala-cli/issues/575

I will also include fixes in a followup PR.